### ISSUE TITLE
Suggestion: Take comments from syntax yamls 

### ISSUE THREAD

**Issue by MarcusE1W** [micro-editor/comment-plugin#13]
My suggestion is to avoid to define the comments per language in this plugin separately.

I don't know how practical it is, but would it be possible to read the file type and corresponding comment style from the yaml file that defines the info for syntax-highlighting?

The comment style should also be in the syntax highlighting rules data structures while micro is running, maybe they can be taken from there?

**Comment by sum01** [comment]
I don't think this is possible unless the plugin gets moved into Micro's included plugins, since the syntax files are compiled into the Micro binary. 

Althgouh I guess if you added Micro as a submodule, then used a YAML parser for Lua, did a bunch of work, it might work...
